scripts/provisioning/setupcmd.py

```diff
@@ -617,3 +618,16 @@ def get_ldap_root_credentials(self):
 
     if enc_rootdn_passwd != None:
       self.rootdn_passwd = s3cipher_obj.decrypt(cipher_key, enc_rootdn_passwd)
+
+  def create_dict_for_BG_delete_account(self):
+    """returns a dictionary required for creating a BG delete account in init and reset phase."""
+    opfileconfstore = S3CortxConfStore(f'yaml://{self.BG_delete_config_file}', 'read_config_idx')
+
+    param_list = ['account_name', 'account_id', 'canonical_id', 'mail', 's3_user_id', 'const_cipher_secret_str', 'const_cipher_access_str']
```

Comment:
see if we can avoid this list. 
we already have key name in the config file.. 
use the same key name as is...

Reply:
@nileshgovande views??
If we have to again read the keys from the config file, we will again have to either call get_all_keys() which will return all the keys from the config file and parse the list or we will have to create a list of parameters by calling the confstore to get the key name for no. of keys.